centreon/www/widgets/src/centreon-widget-resourcestable/src/Listing/translatedLabels.ts

```diff
@@ -83,3 +85,9 @@ export const labelTicketOpenTime = 'Opened on';
 export const labelSubject = 'Subject';
 export const labelOpenTime = 'Open time';
 export const labelCreateticket = 'Create a ticket';
+export const labelCloseATicket = 'Close a ticket';
+export const labelConfirm = 'Confirm';
+export const labelTicketWillBeClosedInTheProvider =
+  'The ticket will be closed on the selected ticket provider';
```

Comment:
```suggestion
  'The ticket will be closed for the selected ticket provider';
```

Reply:
I can't find the string elsewhere, is that normal?

Reply:
yes, can you add it in message.po please?